service/provider/src/main/java/gov/va/vro/service/provider/mas/service/MasProcessingService.java

```diff
@@ -77,12 +77,10 @@ public String processIncomingClaimPresumptiveOffRampClaimCheck(MasAutomatedClaim
     var offRampReasonOptional = getOffRampReasonPresumptiveCheck(payload);
     if (offRampReasonOptional.isPresent()) {
       var offRampReason = offRampReasonOptional.get();
-      Claim claim = toClaim(payload);
       payload.setOffRampReason(offRampReason);
-      claim.setOffRampReason(offRampReason);
-      saveToDbService.setOffRampReason(claim);
-      offRampClaim(payload, offRampReason);
-      return offRampReason;
+      MasProcessingObject mpo = new MasProcessingObject(payload, MasCamelStage.START_COMPLETE);
```

Comment:
We should not need this MasCamelStage.START_COMPLETE anymore. The whole reason was to identify the controller offramps.